xml/System.Xml.Xsl/XslCompiledTransform.xml

```diff
@@ -624,7 +624,7 @@ xslt.Load(typeof(bookOrders));
       </Docs>
     </Member>
     <Member MemberName="Load">
-      <MemberSignature Language="C#" Value="public void Load (string? stylesheetUri, System.Xml.Xsl.XsltSettings settings, System.Xml.XmlResolver stylesheetResolver);" FrameworkAlternate="net-5.0" />
+      <MemberSignature Language="C#" Value="public void Load (string stylesheetUri, System.Xml.Xsl.XsltSettings? settings, System.Xml.XmlResolver? stylesheetResolver);" FrameworkAlternate="net-5.0" />
```

Comment:
Check if expected. Was also expecting XslTransform to be marked obsolete, but it's not.

Reply:
I guess XslTransform is obsolete (according to [remarks](https://docs.microsoft.com/en-us/dotnet/api/system.xml.xsl.xsltransform?view=netcore-3.1#remarks)) but not marked as such: https://github.com/dotnet/runtime/blob/6cf1b8ec012d52880d46fa4773f60ed52ddc9f3d/src/libraries/System.Private.Xml/src/System/Xml/Xslt/XslTransform.cs#L14

Reply:
XslCompiledTransform change looks good: https://github.com/dotnet/runtime/blob/6cf1b8ec012d52880d46fa4773f60ed52ddc9f3d/src/libraries/System.Private.Xml/src/System/Xml/Xslt/XslCompiledTransform.cs#L100